packages/components/src/notice/list.js

```diff
@@ -18,9 +19,11 @@ import Notice from './';
 * @param  {Object}   $0.children  Array of children to be rendered inside the notice list.
 * @return {Object}                The rendered notices list.
 */
-function NoticeList( { notices, onRemove = noop, className = 'components-notice-list', children } ) {
+function NoticeList( { notices, onRemove = noop, className, children } ) {
 	const removeNotice = ( id ) => () => onRemove( id );
 
+	className = classnames( 'components-notice-list', className );
```

Comment:
We're changing how the `className` prop works here. Should we make a note of it in `packages/components/CHANGELOG.md`?

Reply:
d2599e278